exotic.py

```diff
@@ -1646,7 +1656,8 @@ def parse_args():
                     quadLimb = float(quadString)
                     break
                 except:
-                    filterName = input('\nNot valid filter name. Please enter a valid filter name using aavso.org/filters: ')
+                    filterName = input('\nNot valid filter name. Please enter a valid filter name using '
```

Comment:
Ultimately I think pointing them to the EXOFAST website is correct, but we will need to keep this in mind when we ingest data from this version of EXOTIC into CITISENS. We should think about contacting Jason Eastman at EXOFAST to see exactly what wavelengths/passbands correspond to each of the listed filters on EXOFAST.